go/vt/tableacl/tableacl_test.go

```diff
@@ -102,3 +102,7 @@ func checkAccess(configData []byte, tableName string, role Role, t *testing.T, w
 		t.Errorf("got %v, want %v", got, want)
 	}
 }
+
+func allString() string {
+	return GetCurrentAclFactory().AllString()
+}
```

Comment:
this will reduce the test coverage of tableacl package. this is only reason why I keep it...